components/ai_chat/core/browser/engine/engine_consumer_oai.cc

```diff
@@ -47,9 +46,9 @@ using mojom::ConversationTurn;
 
 base::Value::List BuildMessages(
     const mojom::CustomModelOptions& model_options,
-    const std::string& page_content,
+    base::Value::List page_content_messages,
     const std::optional<std::string>& selected_text,
-    const bool& is_video,
+    int max_associated_content_length,
```

Comment:
?

Reply:
whoops, that was leftover from an old refactoring